src/constraints/storage_constraints.jl

```diff
@@ -43,7 +43,8 @@ function add_general_storage_dispatch_constraints(m, p, b; _n="")
     #Constraint (4j): Dispatch from storage is no greater than power capacity
 	@constraint(m, [ts in p.time_steps],
         m[Symbol("dvStoragePower"*_n)][b] >= m[Symbol("dvDischargeFromStorage"*_n)][b, ts]
-    )
+        + m[Symbol("dvStorageToGrid")][ts] # TODO: add "b" index to this decision variable
```

Comment:
Either dvStorageToGrid should be included within dvDischargeFromStorage and subtracted out in the load balance equations, or if it's not included like you've done here then dvDischargeFromStorage needs to be renamed to dvStorageToLoad.